Remotion/Web/Development.WebTesting/WebDriver/Configuration/IBrowserConfiguration.cs

```diff
@@ -20,17 +20,13 @@
 using Remotion.Web.Development.WebTesting.ScreenshotCreation;
 using Remotion.Web.Development.WebTesting.ScreenshotCreation.Annotations;
 using Remotion.Web.Development.WebTesting.Utilities;
-using Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chrome;
-using Remotion.Web.Development.WebTesting.WebDriver.Configuration.InternetExplorer;
 using Remotion.Web.Development.WebTesting.WebDriver.Factories;
 
 namespace Remotion.Web.Development.WebTesting.WebDriver.Configuration
 {
   /// <summary>
   /// Provides configuration needed to initialize and shut down the browser
   /// </summary>
-  /// <seealso cref="ChromeConfiguration"/>

```

Comment:
why removed isntead of adding the new type?

Reply:
I did not see the use in listing implementations here, what would be the use?

Reply:
Yep, please add. Listing the derived types helps with navigating the documentation / getting an overview of the code model while reading the comments.